client/src/pages/HomePage.js

```diff
@@ -0,0 +1,47 @@
+import { useState } from 'react'
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+import LoginForm from '../components/LoginForm'
+import ProfileForm from '../components/ProfileForm'
+
+HomePage.propTypes = {
+  onSubmit: PropTypes.func.isRequired,
+  onLogin: PropTypes.func.isRequired,
+}
+
+export default function HomePage({ onSubmit, onLogin }) {
+  const [entryForm, setEntryForm] = useState(false)
+  return (
+    <>
+      {!entryForm && (
+        <Wrapper>
+          <LoginForm onLogin={onLogin} />
+          <span
+            onClick={() => setEntryForm(!entryForm)}
+            data-testid="modusForm"
+          >
+            or registrate
+          </span>
```

Comment:
This should be a `button` element.
```suggestion
          <button
            onClick={() => setEntryForm(!entryForm)}
          >
            or registrate
          </button>
```

Reply:
jop